drivers/vulkan/vulkan_context.cpp

```diff
@@ -1279,6 +1283,9 @@ Error VulkanContext::_create_physical_device(VkSurfaceKHR p_surface) {
 	//  features based on this query
 	vkGetPhysicalDeviceFeatures(gpu, &physical_device_features);
 
+	// Check required features
+	ERR_FAIL_COND_V_MSG(!physical_device_features.imageCubeArray, ERR_CANT_CREATE, "Your GPU doesn't support image cube arrays.");
```

Comment:
We will have to add a similar check for independant blend. Its supported on 99.12% of devices so should not be a big deal to ensure it is avaialble
```suggestion
	ERR_FAIL_COND_V_MSG(!physical_device_features.imageCubeArray, ERR_CANT_CREATE, "Your GPU doesn't support image cube arrays which are required to use the Vulkan-based renderers in Godot.");
	ERR_FAIL_COND_V_MSG(!physical_device_features.independentBlend, ERR_CANT_CREATE, "Your GPU doesn't support independentBlend which is required to use the Vulkan-based renderers in Godot.");
```

Reply:
done

Reply:
Looks like you missed the second line which checks for ``independentBlend``

Reply:
Ugh, yeah I did, I'm blind :)